baseplate/lib/tracing.py

```diff
@@ -42,3 +44,72 @@ def should_sample(
 
     def get_description(self) -> str:
         return f"RateLimited(fixed rate sampling {self.rps})"
+
+
+# Greenlet tracing utils
+__Greenlet = gevent.Greenlet
+__IMap = gevent.pool.IMap
+__IMapUnordered = gevent.pool.IMapUnordered
+
+
+class Runnable(Protocol):
+    @property
+    def trace_context(self) -> Context: ...
+
+    run: Callable
+
+
+class TracingMixin:
+    def __init__(self: Runnable, *args, **kwargs) -> None:  # type: ignore
```

Comment:
Instead of `type: ignore`, can we do

```diff
diff --git a/baseplate/lib/tracing.py b/baseplate/lib/tracing.py
index 2f6ac41..d5aa671 100644
--- a/baseplate/lib/tracing.py
+++ b/baseplate/lib/tracing.py
@@ -1,5 +1,5 @@
 from collections.abc import Callable, Sequence
-from typing import Optional, Protocol
+from typing import Optional, Protocol, Any

 import gevent.pool
 from opentelemetry import context
@@ -60,7 +60,7 @@ class Runnable(Protocol):


 class TracingMixin:
-    def __init__(self: Runnable, *args, **kwargs) -> None:  # type: ignore
+    def __init__(self: Runnable, *args: Any, **kwargs: Any) -> None:
         self.trace_context = context.get_current()
         super().__init__(*args, **kwargs)
```

This seems to pass mypy for me.

Reply:
Thanks, I wasn't able to figure this one out!